docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -397,6 +397,18 @@ based on your project's build tool:
 
 |===
 
+The default can be overridden by providing an output directory when registering the
```

Comment:
This requires JUnit 5.1 or later (which isn't unreasonable). Could you preface this with something like "If you are using JUnit 5.1 or later,"?

Reply:
Done.